.github/workflows/cd.yml

```diff
@@ -49,9 +55,14 @@ jobs:
         run: |
           cd target/${{ matrix.target }}/release
 
-          strip spt
-          tar czvf spotify-tui-${{ matrix.artifact_prefix }}.tar.gz spt
-          shasum -a 256 spotify-tui-${{ matrix.artifact_prefix }}.tar.gz > spotify-tui-${{ matrix.artifact_prefix }}.sha256
+          strip spt${{ matrix.binary_postfix }}
+          tar czvf spotify-tui-${{ matrix.artifact_prefix }}.tar.gz spt${{ matrix.binary_postfix }}
+          
+          if [[ ${{ runner.os }} == 'Linux' ]]; then
+            shasum -a 256 spotify-tui-${{ matrix.artifact_prefix }}.tar.gz > spotify-tui-${{ matrix.artifact_prefix }}.sha256
+          elif [[ ${{ runner.os }} == 'Windows' ]]; then
+            certutil -hashfile spotify-tui-${{ matrix.artifact_prefix }}.tar.gz sha256 | grep -E [A-Fa-f0-9]{64} > spotify-tui-${{ matrix.artifact_prefix }}.sha256
+          fi
```

Comment:
Looks like this logic is missing the `sha` needed for the macOS binary?

Could do something like this?

```bash
if [[ ${{ runner.os }} == 'Windows' ]]; then
  certutil -hashfile spotify-tui-${{ matrix.artifact_prefix }}.tar.gz sha256 | grep -E [A-Fa-f0-9]{64} > spotify-tui-${{ matrix.artifact_prefix }}.sha256
else
  # Generate sha for both macOS and Linux
  shasum -a 256 spotify-tui-${{ matrix.artifact_prefix }}.tar.gz > spotify-tui-${{ matrix.artifact_prefix }}.sha256
```

Reply:
Good spot! [Here](https://github.com/MCOfficer/spotify-tui/releases/tag/v0.0.4)'s a tag with the latest commit.